### ISSUE TITLE
Unable to install dependencies

### ISSUE THREAD

**Issue by hroussel** [Nogias9x/github-issue-web#1]
## Steps to reproduce

```
➜  github-issue-web git:(master) npm install

> github-issue-web@4.0.0 preinstall /Users/hroussel/Downloads/github-issue-web
> npm run npmcheckversion

npm ERR! missing script: npmcheckversion

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/hroussel/.npm/_logs/2019-06-03T07_44_13_924Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! github-issue-web@4.0.0 preinstall: `npm run npmcheckversion`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the github-issue-web@4.0.0 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/hroussel/.npm/_logs/2019-06-03T07_44_13_963Z-debug.log
➜  github-issue-web git:(master) npm --version
6.4.1
```

**Expected behavior**

Npm to install dependencies

## Versions

- Node/NPM: v10.15.3/6.4.1
- OS: OSX Mojave

**Comment by Nogias9x** [comment]
I'm looking into this immediately @hroussel

**Comment by Nogias9x** [comment]
Thank you for your patience @hroussel,

I've fixed it in commit [741c03b](https://github.com/Nogias9x/github-issue-web/commit/741c03be13d1b955a22fdaa9e7feacdb94d0850f).
The cause is a script which relates to `npm install` was missing

Could you please help me pull `master` branch and check it again?

**Comment by hroussel** [comment]
It works now!
